imageio/imageio-pict/src/main/java/com/twelvemonkeys/imageio/plugins/pict/PICTImageReader.java

```diff
@@ -2628,7 +2627,7 @@ public int getHeight(int pIndex) throws IOException {
 
     public Iterator<ImageTypeSpecifier> getImageTypes(int pIndex) throws IOException {
         // TODO: The images look slightly different in Preview.. Could indicate the color space is wrong...
-        return Collections.singletonList(
+        return Arrays.asList(
```

Comment:
You should probably keep the `Collections.singletonList` here, unless you have a  specific reason to change it?